EnglishFilter/sections/foreign.txt

```diff
@@ -807,6 +807,11 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/41555
 ||m.pianba.tv/ad/
+! https://github.com/AdguardTeam/AdguardFilters/issues/41968
+technews.tw#%#AG_abortInlineScript('/navigator.userAgent/', 'jQuery');
+technews.tw#%#//scriptlet("abort-current-inline-script", "jQuery", "/navigator.userAgent/")
+!+ PLATFORM(ext_android_cb, ext_safari, ios)
+@@||technews.tw/images/blockadblock/adserver.js
```

Comment:
It's just a bait, so I think that hint (`!+ PLATFORM(ext_android_cb, ext_safari, ios`) is not needed in this case:)